src/app/fdctl/run/topos/fd_firedancer.c

```diff
@@ -73,7 +76,6 @@ fd_topo_firedancer( config_t * _config ) {
   fd_topob_wksp( topo, "gossip_voter" );
   fd_topob_wksp( topo, "voter_gossip" );
   fd_topob_wksp( topo, "voter_dedup"  );
-  fd_topob_wksp( topo, "poh_replay"   );
```

Comment:
In #3013, Liam removed `POH_IN_IDX` from the replay tile, so this link seems to be obsolete. Need to confirm with Liam whether this is correct.